service/src/main/resources/templates/ehr_narrative_statement_template.mustache

```diff
@@ -4,5 +4,18 @@
         <statusCode code="COMPLETE" />
         <availabilityTime value="{{availabilityTime}}" />
         <text>{{comment}}</text>
+        {{#hasReference}}
+        <reference typeCode="REFR">
+            <referredToExternalDocument classCode="DOC" moodCode="EVN">
+                <id root="{{narrativeStatementId}}"/>
+                <code code="25821000000100" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Original Text document">
+                    <originalText>Record Attachment</originalText>
+                </code>
```

Comment:
Where did this code come from? In the mapping sheet it seems that 

`<code code="37251000000104" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Other digital signal">`

is used instead

Reply:
I took this from the example xml attached to the ticket, I'll use the example in the mapping spreadsheet instead